control/mttrie.go

```diff
@@ -110,15 +107,14 @@ func (m *MTTrie) RemoveMetric(mt metricType) {
 	if a != nil {
 		for v, x := range a.mts {
 			if mt.Version() == x.Version() {
-				// Delete the metric from the node
+				// delete this metric from the node
```

Comment:
> Why change from children->descendants

I changed children to descendants because it was confusing - it's gathering all descendants, not only children (direct descendants). Now it becomes consistent through this file and children mean direct descendants. Before it was not clear.

> change the capitalization of a comment
Please take a look on another comments in this file and how they look:
- https://github.com/intelsdi-x/snap/blob/master/control/mttrie.go#L132
- https://github.com/intelsdi-x/snap/blob/master/control/mttrie.go#L178

So, I decided to change  "Delete" > "delete" to keep consistence with the rest of comments.